app/upgrade_cudos_distribution.go

```diff
@@ -396,12 +396,12 @@ func withdrawGenesisDistributionRewards(app *App, genesisData *GenesisData, cudo
 
 	communityBalance, _ := genesisData.distributionInfo.feePool.communityPool.TruncateDecimal()
 	remainingBalance := distributionModuleAccount.balance.Sub(communityBalance)
-	app.Logger().Info("Remaining dist balance", "amount", remainingBalance.String())
+	app.Logger().Info("cudos merge: remaining dist balance", "amount", remainingBalance.String())
 
 	// TODO: Write to manifest?
 	err = checkTolerance(remainingBalance, maxToleratedRemainingDistributionBalance)
 	if err != nil {
-		return fmt.Errorf("Remaining distribution balance %s is too high", remainingBalance.String())
+		return fmt.Errorf("remaining distribution balance %s is too high", remainingBalance.String())
```

Comment:
```suggestion
		return fmt.Errorf("cudos merge: remaining distribution balance %s is too high", remainingBalance.String())
```

Reply:
cudos merge prefix is added to error in `CudosMergeUpgradeHandler`